### PR TITLE
Feature/welcome page text+content

### PR DESCRIPTION
**Pull request by VaishaliGovindaraj**
Hi all,

I have added the welcome page content and added styling for desktop and smaller screen.
Please check and let me know  your comments.

### CODE REVIEW

File: scripts/index.js [hunk 1]
```diff
@@ -1,9 +1,83 @@
...
+        "images/index/welcome_page/explore_zoo/playarea_3.jpg",
+        "images/index/welcome_page/explore_zoo/playarea_4.jpg"
+    ],
+    "Please note that DjurparkExploring a zoo is a fascinating journey into the world of wildlife and conservation. From lush green aviaries to aquatic exhibits, every corner unveils unique habitats teeming with life. Visitors can marvel at majestic big cats prowling their enclosures, playful otters gliding through water, or primates showcasing their agility and intelligence. Interactive zones like petting zoos provide hands-on experiences, while informative signage and guided tours offer insights into animal behavior and conservation efforts. Nocturnal exhibits reveal the wonders of creatures active at night, and live feeding sessions let keepers share captivating stories about diets and habits. Relaxing at shaded picnic spots or zoo cafes adds a refreshing touch, making the visit both enjoyable and educational. A day at the zoo is more than entertainment—it's an inspiring opportunity to connect with and protect our planet’s incredible biodiversity. does not have luggage storage facilities i.e. lockers. If you have large bags, please chat."
```
Comment (Lvhacker1) [inline scripts/index.js]: DjurparkExploring?

**Review by Ankitasingh88 — COMMENTED** [review]
Can we add a block in mobile view instead of current view?

**Review by Ankitasingh88 — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -10,21 +11,13 @@
...
-            <nav class="nav_menu">
-                <a href="#">Home</a>
-                <a href="#">Reptiles</a>
-                <a href="#">Mammals</a>
-                <a href="#">Reptiles</a>
-                <a href="#">About-Us</a>
-            </nav>
-        </header>
-    </div>
+    <nav class="nav_menu">
```
Comment (Lvhacker1) [inline index.html:16]: Why have you made changes here?

**Review by Lvhacker1 — CHANGES_REQUESTED** [review]

**Review by PriyeshIllam — APPROVED** [review]
Looking good to me

File: scripts/index.js (hunk 1 above)
Comment (VaishaliGovindaraj) [reply scripts/index.js]: there should be a space in between, will add that

File: index.html (hunk 2 above)
Comment (VaishaliGovindaraj) [reply index.html:16]: for mobile  view layout i needed to remove it since the contents were overflowing

File: scripts/index.js (hunk 1 above)
Comment (Lvhacker1) [reply scripts/index.js]: Please make sure to change the name to SafariPark as well.

File: scripts/index.js [hunk 3]
```diff
@@ -34,7 +34,7 @@ let closeEncounters = new InfoText(
...
-    "Experience the thrill of getting up close and personal with some of the world’s most fascinating animals through our exclusive Close Encounter programs! At Te Nukuao Wellington Zoo, we provide a unique opportunity for visitors to meet and interact with a variety of creatures in a safe and controlled environment. Imagine hand-feeding giraffes, walking alongside curious meerkats, or even spending time with majestic big cats under the guidance of our expert keepers. These encounters are designed to deepen your connection with wildlife and foster a greater appreciation for conservation efforts."
+    "Experience the thrill of getting up close and personal with some of the world’s most fascinating animals through our exclusive Close Encounter programs! At Djurpark, we provide a unique opportunity for visitors to meet and interact with a variety of creatures in a safe and controlled environment. Imagine hand-feeding giraffes, walking alongside curious meerkats, or even spending time with majestic big cats under the guidance of our expert keepers. These encounters are designed to deepen your connection with wildlife and foster a greater appreciation for conservation efforts."
```
Comment (Lvhacker1) [inline scripts/index.js]: SafariPark :)

File: scripts/index.js (hunk 1 above)
Comment (VaishaliGovindaraj) [reply scripts/index.js]: Okay sure

File: scripts/index.js [hunk 4]
```diff
@@ -13,7 +13,7 @@ let planYourDay = new InfoText(
...
+    "SafariPark is open from 9:30-5 pm, all year except Christmas Day.Child entry is for 3-14 year olds. Free entry for children younger than 3 years old.Under 14s need to have an adult with them.Please bring your ID, as we offer discounts to students, Community Services card holders, and Gold Card holders.Caregivers of disabled persons are given complimentary entry to the Zoo upon presentation of suitable supporting documentation or identification.Please note that Te Nukuao Wellington Zoo does not have luggage storage facilities i.e. lockers.  If you have large bags, please chat with one of our Visitor Advisors, who may be able to store them in the front office during your visit.");
```
Comment (Lvhacker1) [inline scripts/index.js:15]: Great job on the update!

**Review by Lvhacker1 — APPROVED** [review]
Great job on the quick change!

File: scripts/index.js (hunk 4 above)
Comment (Lvhacker1) [reply scripts/index.js:15]: Sorry, you need to update one more thing.

Comment (Lvhacker1) [reply scripts/index.js:15]: SafariPark is open from 9:30 AM to 5:00 PM, all year except Christmas Day. Child entry is for 3-14 year olds. Free entry for children younger than 3 years old. Children under 14 need to have an adult with them. Please bring your ID, as we offer discounts to students, Community Services card holders, and Gold Card holders. Caregivers of disabled persons are given complimentary entry to the zoo upon presentation of suitable supporting documentation or identification. Please note that SafariPark does not have luggage storage facilities (i.e., lockers). If you have large bags, please chat with one of our Visitor Advisors, who may be able to store them in the front office during your visit.

**Review by Lvhacker1 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**